Source/Utils/WarpXMovingWindow.cpp

```diff
@@ -348,6 +348,43 @@ WarpX::shiftMF (MultiFab& mf, const Geometry& geom, int num_shift, int dir,
     }
 }
 
+void
+WarpX::ShiftGalileanBoundary ()
+{
+    Real cur_time = t_new[0];
+    WarpX:: time_of_last_gal_shift;
```

Comment:
The above statement does not seem to do anything. Could you remove it?

Reply:
Done in commit 3d584488063.